README.md

```diff
@@ -46,7 +60,11 @@ This tool is capable of reconstructing damaged WAVE and AIFF headers. This will
     * If you used a sample rate of 48 kHz, you have to add `-s 48000` before `audio`.
     * If the files have two channels (i.e. stereo instead of mono), then you have to add `-c 2` before `audio`.
     * See section *Restoring Damaged WAVE/AIFF File Headers* for more details.
-
+14. Install Pyqt5
+    ```
+    pip install pyqt5
```

Comment:
This does not work on Mac. The command is called `pip3` instead of `pip`. If Python was installed using `homebrew`, this does not work at all. Then users have to install Python-Qt5 using

```
brew install pyqt5
```